packages/klevu-ui/src/components/klevu-quicksearch/klevu-quicksearch.tsx

```diff
@@ -351,6 +377,10 @@ export class KlevuQuicksearch {
     this.currentViewPortSize = event.detail
   }
 
+  #isNoResultsPage() {
```

Comment:
This could be just variable in start of the `render()` method? 
```
render() {
  const isNoResultsPage = !!this.#searchTerm
```

Reply:
changed to variable